src/state/store.js

```diff
@@ -79,61 +80,48 @@ store.reset = () => {
   });
 };
 
-function getActiveProductConfig() {
+export function getActiveProductConfig() {
```

Comment:
It doesn't look like this is used externally. Is the plan to use it externally in the future?

Reply:
ah, I think I was going to use this somewhere, but decided against it. Will remove